ComSemApp/student/urls.py

```diff
@@ -22,4 +22,5 @@
     url(r'^course/(?P<course_id>[0-9]+)/speakingpractice/results/$', views.SpeakingPracticeResultsView.as_view(), name='speaking_practice_results'),
     url(r'^course/(?P<course_id>[0-9]+)/speakingpractice/instructions/$', views.SpeakingPracticeInstructionsView.as_view(), name='speaking_practice_instructions'),
     url(r'^course/(?P<course_id>[0-9]+)/speakingpractice/save$', views.SpeakingPracticeAttemptCreateView.as_view(), name='speaking_practice_submit'),
+       url(r'^course/(?P<course_id>[0-9]+)/speakingpractice/request$', views.SpeakingPracticeTeacherReviewRequestCreateView.as_view(), name='speaking_practice_teacher_request')
```

Comment:
Mainly a preference thing, but I feel `'speaking_practice_review_request'` might be a better name than `'speaking_practice_teacher_request'`